styles.css

```diff
@@ -0,0 +1,89 @@
+
+@media screen and (min-width: 375px) {
+  /* universal formatting */
+
+  *,
+  *::before,
+  *::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #fff;
+  }
+
+  html {
+    scroll-behavior: smooth;
+  }
+
+  .main-header {
+    height: 5rem;
+    padding: 24px;
+    background-color: #fff;
+  }
+
+  .info {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .info a {
+    text-decoration: none;
+    color: #6070ff;
+    font-size: 20px;
+    font-weight: 700;
+    text-transform: uppercase;
+  }
+
+  .headlines {
+    background-image: url("assets/images/header-shapes\ mobile.png");
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-top: -10px;
+    background-color: #fff;
+    border-radius: 0 0 0 40px;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .headlines .about {
+    padding: 24px;
+  }
+
+  .headlines .intro {
+    font-size: 40px;
+    font-weight: 700;
+  }
+
```

Comment:
- [ ] Your design looks great :+1: But please make sure that you give this intro a color of `#172B4D` to match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=oarhayyFqhw65rmQ-0). Currently, you have the default color `black` which does not match what is in the [Figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=oarhayyFqhw65rmQ-0)
  
  ### Your Design
  ![image](https://user-images.githubusercontent.com/92303891/227045578-bdddf178-8a60-4edd-9dfa-28b626c08346.png)
  
  ### Figma design
  ![image](https://user-images.githubusercontent.com/92303891/227045878-c753aacd-f90e-4abc-b2e8-b7bdb5325ba4.png)
  
  
  `